test/integration/backward_compatible/parallel/serialization/DefaultSerializersLiveTest.js

```diff
@@ -179,6 +204,18 @@ describe('DefaultSerializersLiveTest', function () {
         expect(actualValue).to.deep.equal([1, 2, 3]);
     });
 
+    it('should deserialize empty ArrayList', async function () {
+        TestUtil.markClientVersionAtLeast(this, '4.2');
+        const script = 'var map = instance_0.getMap("' + map.getName() + '");\n' +
+            'var list = new java.util.ArrayList();\n' +
+            'map.set("key", list);\n';
```

Comment:
Wouldn't using a backtick string make these lines more clear? I think the same approach can be used for other similar cases in this file.

Reply:
It was this way before. Sure we can do that. It's a matter of style

Reply:
done in https://github.com/hazelcast/hazelcast-nodejs-client/pull/1197/commits/5025dc239356ec1d46e99e2bea640f8c4cb9410b

We can also make the entire script as one backtick string, I don't see a lot of value in doing it.